README.md

```diff
@@ -288,11 +288,11 @@ and will extract everything into one `~/.shiv` directory unless otherwise specif
 `ducktools-env` will create a separate environment for each unique set of requirements
 for temporary environments by matching specification.
 
-### PEX ###
+### Pex ###
 
 `pex` provides an assortment of related tools for developers alongside a `.pex` bundler.
-It doesn't (to my knowledge) have support for inline script metadata and it makes `.pex` files
```

Comment:
That is convention only. `.pex`, `.pyz`, `.zip`, no extension - it doesn't actually matter at all - except maybe on Windows, which Pex only partially supports, and not officially.

Reply:
FWIW, Pex also supports embedding Python. The gateway is `--scie {eager,lazy}` with a bunch of other associated options. That will net you a native executable that embeds Python Build Standalone as the Python.